rs/recovery/src/util.rs

```diff
@@ -4,10 +4,31 @@ use crate::{
 };
 
 use ic_base_types::{NodeId, PrincipalId, SubnetId};
+use serde::{Deserialize, Serialize};
 use slog::{o, Drain, Logger};
+use std::net::{IpAddr, Ipv6Addr};
 use std::{future::Future, path::Path, str::FromStr};
 use tokio::runtime::Runtime;
 
+pub const IPV6_LOOPBACK_ADDRESS: Ipv6Addr = Ipv6Addr::new(0, 0, 0, 0, 0, 0, 0, 1);
```

Comment:
Actually, I think there already exists `Ipv6Addr::LOCALHOST` which we could use instead